pkg/util/ecs/metadata/v3or4/types.go

```diff
@@ -7,17 +7,24 @@ package v3or4
 
 // Task represents a task as returned by the ECS metadata API v3 or v4.
 type Task struct {
-	ClusterName           string             `json:"Cluster"`
-	Containers            []Container        `json:"Containers"`
-	KnownStatus           string             `json:"KnownStatus"`
-	TaskARN               string             `json:"TaskARN"`
-	Family                string             `json:"Family"`
-	Version               string             `json:"Revision"`
-	Limits                map[string]float64 `json:"Limits,omitempty"`
-	DesiredStatus         string             `json:"DesiredStatus"`
-	LaunchType            string             `json:"LaunchType,omitempty"` // present only in v4
-	ContainerInstanceTags map[string]string  `json:"ContainerInstanceTags,omitempty"`
-	TaskTags              map[string]string  `json:"TaskTags,omitempty"`
+	ClusterName             string             `json:"Cluster"`
+	Containers              []Container        `json:"Containers"`
+	KnownStatus             string             `json:"KnownStatus"`
+	TaskARN                 string             `json:"TaskARN"`
+	Family                  string             `json:"Family"`
+	Version                 string             `json:"Revision"`
+	Limits                  map[string]float64 `json:"Limits,omitempty"`
+	DesiredStatus           string             `json:"DesiredStatus"`
+	LaunchType              string             `json:"LaunchType,omitempty"` // present only in v4
+	ContainerInstanceTags   map[string]string  `json:"ContainerInstanceTags,omitempty"`
+	TaskTags                map[string]string  `json:"TaskTags,omitempty"`
+	EphemeralStorageMetrics map[string]int64   `json:"EphemeralStorageMetrics,omitempty"`
```

Comment:
Do we need this field ?

Reply:
Yes it's one of the attributes https://github.com/aws/aws-sdk-go-v2/blob/service/ecs/v1.41.1/service/ecs/types/types.go#L3691